extensions/lifecycle/tasks/LifecycleUpdateTransitionTask.js

```diff
@@ -98,6 +115,20 @@ class LifecycleUpdateTransitionTask extends BackbeatTask {
         this.gcProducer.publishActionEntry(gcEntry, done);
     }
 
+    _checkAndGarbageCollectLocation(entry, locations, log, done) {
+        return async.series([
+            next => this._checkDate(entry, log, next),
+            next => this._garbageCollectLocation(entry, locations, log, next),
+        ], err => {
+            if (err && err.statusCode === 412) {
+                log.info('Object was modified after transition had begun ' +
```

Comment:
minor: Instead of "object was not deleted" I would refer to the data location, e.g. "data location was not deleted", not to confuse with deletion of the zenko object itself. "Object was modified" looks fine as it can apply to the zenko object as well as the underlying cloud object.